workshops/management/commands/check_for_workshop_websites_updates.py

```diff
@@ -0,0 +1,244 @@
+from collections.abc import Iterable
+import datetime
+from functools import partial
+import json
+
+from django.core.management.base import BaseCommand
+from github import Github
+import requests
+from rest_framework.utils.encoders import JSONEncoder
+
+from workshops.models import Event
+from workshops.util import (
+    fetch_event_tags,
+    parse_tags_from_event_website,
+    WrongWorkshopURL,
+)
+
+
+def datetime_match(string):
+    """Convert string date/datetime/time to date/datetime/time."""
+    try:
```

Comment:
~~~
for format in self.TIME_FORMATS:
  try:
    v = datetime.datetime.strptime(string, format)
    break
  except ValueError:
    pass
~~~

or something like that?